manticore/core/state.py

```diff
@@ -311,43 +311,6 @@ def record_branches(self, targets):
             except KeyError:
                 self.branches[key] = 1
 
-    def generate_inputs(self, workspace, generate_files=False):
```

Comment:
Instead of removing it consider moving this ->  https://github.com/trailofbits/manticore/blob/master/manticore/core/workspace.py#L389
Or is it already there?

(Then in another PR remove that class)